modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -509,8 +618,25 @@ m|INFORMATION
 m|PERFORMANCE
 |===
 
-The `CodeGenerationFailed` notification is created when it is not possible to generate a code for a query, for example, when the query is too big.
-To find more information about the specific query, see the stack trace in the _debug.log_ file.
+======
+[.include-with-GQLSTATUS-code]
+======
+
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|01N40
+|StatusDescription
+a|info: runtime unsupported.
+The query cannot be executed `runtime=pipelined operatorEngine=compiled expressionEngine=compiled`, `runtime=pipelined operatorEngine=interpreted expressionEngine=compiled` is used. Cause: `method too big`.
```

Comment:
```suggestion
The query cannot be executed with `preparser_input1`, `preparser_input2` is used. Cause: `$msg`.
```

This is the general status description and not an example (the reason there is no example is this warning is kind of internal and only occur for things like memory errors in runtime so it is a bit hard to make a good example)